poolmanager/server/controlplane/plugin/noop.go

```diff
@@ -1,56 +1,48 @@
-/*
-	Interface between the Node Pool Manager and the Control Plane
-*/
-
-package cp
+/**
+ * Dummy implementation for the controlplane that just adds delays
+ */
+package main
```

Comment:
package noop? package noopcp? I do not see a main in this package checked in, at least.

Reply:
It's a requirement of buildmode=plugin, the symbols are only exported from the main package